tensortrade/binance/trader.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+# File:        tensortrade/binance/trader.py
+# By:          Samuel Duclos
+# For:         Myself
+# Description: This file handles raw trading (buying and selling) 
+#              for a given pair.
+
+from .pair_info import Binance_pair_info
```

Comment:
@carlogrisetti do you think we should have this trader concept made more abstract?

Reply:
It will be merged to OMS correctly. This is before outputs are supported for live trading

Reply:
> @carlogrisetti do you think we should have this trader concept made more abstract?

I totally agree about abstraction

Reply:
Sorry, I have no idea what your last comment means.

Reply:
> Sorry, I have no idea what your last comment means.

I will have to rewrite some OMS classes to better abstract the bot and enable live trading, but until I close the feedback loop by enabling outputs (trades), this is self-contained until I review what differs (and may have been forgotten) in the OMS.